test/service_instance_and_binding_test/context_signature/context_signature_test.go

```diff
@@ -101,4 +101,26 @@ var _ = Describe("context signature verification tests", func() {
 			})
 		})
 	})
+
+	Context("Invalid private key", func() {
+		instanceID := "signed-ctx-instance-invalid"
+		var provisionFunc func() string
+		BeforeEach(func() {
+			ctx = common.NewTestContextBuilderWithSecurity().WithEnvPostExtensions(func(e env.Environment, servers map[string]common.FakeServer) {
+				e.Set("api.osb_rsa_private_key", "invalidKey")
+			}).Build()
+			ctxSetup()
+		})
+
+		It("request should be sent without the signature", func() {
```

Comment:
The "context" and "it"  should form a readable story. 
eg. 
context("when the private key is invalid", func(){
    it("should not fail the request"/"should send the request without the signature")
})
then when you read it in the logs you can easily understand what is being tested.
In contrast to "Invalid private key. it request should be sent without the signature"